openlibrary/core/vendors.py

```diff
@@ -419,10 +421,12 @@ def _get_betterworldbooks_metadata(isbn):
             price = _price
             qlt = 'new'
 
-    return betterworldbooks_fmt(isbn, qlt, price)
+    market_price = market_price and '$' + market_price[0]
```

Comment:
Is _and_ really what we want here?
```python
>>> for market_price in "123.45 12.34 1.23 12 1".split() + [""]:
...     print(market_price, market_price and '$' + market_price[0])
...     
123.45 $1
12.34 $1
1.23 $1
12 $1
1 $1
```

Reply:
market price should be a list (e.g. ['12.34']) so this would be:
```
>>> ['12.34'] and '$' + ['12.34'][0]
'$12.34'
```